dev-docs/vNext/motion-design-export-rfc.md

```diff
@@ -0,0 +1,44 @@
+# RFC: Integration - 3D Motion Video Editing Software
+
+* Authors: Chris Gervang
+* Date: October, 2020
+* Status: **Draft**
+
+## Overview
+
+This proposal outlines a series of features to enable faster and more effective collaboration between engineering and motion design teams. This can be achieved by exporting video assets and data to sync compositions.
+
+## Problem
+
+Currently motion designers can import videos of visualizations created with vis.gl into 3D composition software, such as AfterEffects or 3DS max, but in doing so the visualization is flattened and all metadata about the motion of the scene is lost. This limits a motion designers ability to advance design tools, and also limits an engineers ability to integrate real data sets in edited videos.
+
+Motion designers currently either recreate a rendition of an engineers data, or spend additional effort with the engineer to polish the appearance in code - either way the result is often prohibitively time consuming, and the work is highly specialized.
+
+## Proposal
+
+Export data from the vis.gl ecosystem in a native format for popular motion design software.
```

Comment:
List a few such formats and add references to documentation?

If (part of) that format is in textual form, perhaps show a snippet to illustrate?

Sketch out some "builder API" that covers the information needed by these formats?